core/bbs.js

```diff
@@ -85,23 +85,18 @@ function main() {
                     //  then it's a fatal error
                     //
                     if (err) {
+                        errorDisplayed = true;
+                        console.error(`Configuration error: ${err.message}`); //  eslint-disable-line no-console
+
                         if ('ENOENT' === err.code) {
-                            if (configPathSupplied) {
-                                console.error(
-                                    'Configuration file does not exist: ' + configFile
-                                );
-                            } else {
-                                configPathSupplied = null; //  make non-fatal; we'll go with defaults
-                            }
-                        } else {
-                            errorDisplayed = true;
-                            console.error(`Configuration error: ${err.message}`); //  eslint-disable-line no-console
-                            if (err.hint) {
-                                console.error(`Hint: ${err.hint}`);
-                            }
-                            if (err.configPath) {
-                                console.error(`Note: ${err.configPath}`);
-                            }
+                            console.error("\nConfiguration file does not exist: '" + configFile + "'\n\nIf this is a new installation please run './oputil.js config new' from the enigma-bbs directory");
```

Comment:
super nit in case you work on this any more before merged: consistent `{configFile}` style here